web/react/components/access_history_modal.jsx

```diff
@@ -57,6 +57,10 @@ module.exports = React.createClass({
                 newDate = (<div> {currentHistoryDate.toDateString()} </div>);
             }
 
+            if (!currentAudit.session_id && currentAudit.action.replace("/api/v1", "") === "/users/login") {
```

Comment:
I did it to keep consistent with how the URL is shown on screen to the user below that but you're right it (and the other use of it) would break on a move to v2.  I'll come it with something better for both, good catch.